packages/dates/src/date-time-utils/DateTimeUtils.ts

```diff
@@ -73,9 +73,10 @@ export function parseDate(
   input: NullableDateInput,
   { format }: DateConfig = defaultDateConfig,
 ): DateTime {
-  if (input instanceof DateTime) {
-    const { defaultZone } = Settings;
+  const { defaultZone } = Settings;
+  Settings.defaultLocale = 'en-US';
```

Comment:
Can we move it into date config file. So whenever library initilialized, it will set default locale?